src/tree/rules/RulesTreeItem.ts

```diff
@@ -83,7 +64,7 @@ export class RulesTreeItem extends AzExtParentTreeItem {
         const resourceGroupName = getResourceGroupFromId(this.apiCenter.id);
         const apiCenterService = new ApiCenterService(this.parent?.subscription!, resourceGroupName, this.apiCenter.name);
 
-        const { value } = await apiCenterService.exportRuleset();
+        const { value } = await apiCenterService.exportRuleset(this.configName);
```

Comment:
what will be if the configName is null?

Reply:
I do not meet with that scenario. But it happens, the `createTreeItemsWithErrorHandling` will show error for this specifc invalid node.